classification/scripts/transforms.py

```diff
@@ -0,0 +1,10 @@
+import albumentations as A
+from albumentations.pytorch import ToTensorV2
+
+
+def get_default_transforms(image_size=224):
+    return A.Compose([
+        A.Resize(image_size, image_size),
+        A.Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225]),
```

Comment:
Where are those numbers from? Is it from ImageNet or you calculated from the dataset?